etc/chromeos/lib/installs.sh

```diff
@@ -46,13 +49,23 @@ install_rust_and_utilities() {
 }
 
 install_pyenv() {
-    echo "Install pyenv and set python version to 3.7.10"
+    echo "Install pyenv and set python version to ${PYENV_PYTHON_VERSION}"
     sudo apt-get install -y make libssl-dev zlib1g-dev libbz2-dev \
         libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
         xz-utils tk-dev libffi-dev liblzma-dev
+
+    # Check if pyenv directory exists and delete it so we can have a clean.
```

Comment:
I made the choice here to delete the old directory, but it may make sense in the future to switch to if installed pyenv install $VERSION && pyenv global $VERSION